examples/function-values/function_values.md

```diff
@@ -1,7 +1,12 @@
 # Function values
 
-Functions are values and work as closures. Function type is a separate basic type. Anonymous function and type syntax look like function definition without the name.
+In Ballerina, a function is also a value implying that it can be stored in variables and passed to or returned from the functions.  A function value can be executed by calling it. A value cannot be assigned to the function identifier.
```

Comment:
What do u mean by `A value cannot be assigned to the function identifier`?

Reply:
```
function A() returns int {
    return 1;
}

public function main() {
    A = function () returns int => 2;
}
```
I'm referring to this case. If not needed we can remove

Reply:
@gimantha Shouldn't this be a compilation error? currently gives no error

Reply:
This should be a compilation error. This fixed this issue https://github.com/ballerina-platform/ballerina-lang/pull/38430

Reply:
Ah okay. How about `A function value cannot be assigned to a defined function name`

Reply:
Updated eb4f4834527feca00a652067472086d0085472b7